sos/report/__init__.py

```diff
@@ -398,7 +398,6 @@ def get_tape_devs(self):
             self.soslog.error("Could not get tape device list: %s" % err)
             return []
 
-
```

Comment:
It's usually better to put clean ups (like non functional whitespace changes) into their own patches rather than tucking them away in another functional commit.